app.py

```diff
@@ -0,0 +1,33 @@
+# Commented out IPython magic to ensure Python compatibility.
+# %cd /spark-data/etl
+
+# %run 'load_data.ipynb'
+# %run 'verify_data.ipynb'
+# %run 'data_cleaning.ipynb'
+# % run 'customer_purchase_behaviour_analysis.ipynb'
```

Comment:
Remove commented code if unused

Reply:
this is because of the .py and .ipynb file
this comments are used in ipynb file but in file these are not resolved that's why i commented it